net-misc/openntpd/openntpd-6.8_p1.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd
+
+MY_P="${P/_p/p}"
+
+DESCRIPTION="Lightweight NTP server ported from OpenBSD"
+HOMEPAGE="http://www.openntpd.org/"
+SRC_URI="mirror://openbsd/OpenNTPD/${MY_P}.tar.gz"
+
+LICENSE="BSD GPL-2"
```

Comment:
By the way there's a catch to this combination, but as you're not the maintainer of this package, I won't have you do the required detective work for it.

But since it's also a quiz question, you should ;)
(ask your mentor about it)